README.md

```diff
@@ -41,13 +44,13 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
 - [Installation](#installation)
 - [Getting Started](#getting-started)
 - [Support + Feedback](#support--feedback)
-- [What is Fingerprint Pro](#what-is-fingerprint-pro)
 - [License](#license)
 
 ## Documentation
 
 This library uses [Fingerprint Pro](https://fingerprint.com/github/) under the hood, you can view the document for the core technology.
-- [Documentation](https://dev.fingerprint.com/docs)
+- To learn more about the Fingerprint Pro read our [product documentation](https://dev.fingerprint.com/docs).
+- To learn more about this SDK, there is [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
```

Comment:
```suggestion
- To learn more about this SDK, there is a [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
```

Reply:
Also, is it just me or does https://fingerprintjs.github.io/fingerprintjs-pro-spa/ give you a 404 now?